src/protocol/models.rs

```diff
@@ -167,7 +178,8 @@ impl GetAmountOutResult {
 }
 
 #[derive(Debug, Clone)]
-pub struct BlockUpdate {
+pub struct Update {
+    pub marker: u64,
```

Comment:
Can you explain the reasoning here? Why did you do this change? What exactly is `marker` I assume it is a timestamp if we don't have a block number? If that is the case what would block number be set to?